src/inmanta/app.py

```diff
@@ -41,11 +41,12 @@
 import time
 import traceback
 import typing
+from argparse import ArgumentParser
 from asyncio import ensure_future
 from configparser import ConfigParser
 from threading import Timer
 from types import FrameType
-from typing import Any, Callable, Coroutine, Dict, Optional
+from typing import Any, Callable, Coroutine, Dict, Optional, Sequence
```

Comment:
`Sequence` should be used from `collections.abc` as `abc.Sequence`. Same goes for some of the others in here but let's do it right at least for any imports we add.